src/handlers/createProduct/handler.ts

```diff
@@ -0,0 +1,67 @@
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { v4 as uuidv4 } from 'uuid';
+import { generateHttpResponse } from '../../utils/lambda';
+import { StatusCode } from '../../types/types';
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, TransactWriteCommand } from "@aws-sdk/lib-dynamodb";
+
+const dynClient = DynamoDBDocumentClient.from(
+  new DynamoDBClient({ region: "eu-west-1" })
+);
+
+export const createProduct = async (event: APIGatewayProxyEvent) => {
+    try {
+
+        if (event.body === null) {
```

Comment:
Please use `zod` library to validate request

Reply:
Done